.buildkite/pipeline.yml

```
@@ -0,0 +1,20 @@
+agents:
+  queue: k8s-builders
+
+steps:
+  - label: make lint
+    key: lint
+    command: make lint
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - json-key: .
+              secret-id: devex/prod/buildkite/integration_provider_secret
```

i've copied the secret value over to a new secret name that adheres to our sdlc secret naming convention:

```suggestion
            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
```
also, no need for `json-key` if the value is `.` because that is the default value assumed.

---

thanks!